src/main/webapp/app/admin/statistics/statistics.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { SERVER_API_URL } from 'app/app.constants';
+import { SpanType } from 'app/entities/statistics.model';
+
+@Injectable({ providedIn: 'root' })
+export class StatisticsService {
+    private resourceUrl = SERVER_API_URL + 'api/management/statistics/';
+
+    constructor(private http: HttpClient) {}
+
+    /**
+     * Sends a GET request to retrieve the amount of logged in users in the last *span* days
+     */
+    getloggedUsers(span: number): Observable<number> {
+        const params = new HttpParams().set('span', '' + span);
+        return this.http.get<number>(`${this.resourceUrl}users`, { params });
+    }
+
+    /**
+     * Sends a GET request to retrieve the amount of users with an submission in the last *span* days
+     */
+    getActiveUsers(span: number): Observable<number> {
+        const params = new HttpParams().set('span', '' + span);
+        return this.http.get<number>(`${this.resourceUrl}activeUsers`, { params });
+    }
+
+    /**
+     * Sends a GET request to retrieve the amount of submissions made in the last *span* days
+     */
+    getTotalSubmissions(span: SpanType): Observable<number[]> {
+        const params = new HttpParams().set('span', '' + span);
+        return this.http.get<number[]>(`${this.resourceUrl}submissions`, { params });
+    }
+
+    /**
+     * Sends a GET request to retrieve the amount of released exercises in the last *span* days
+     */
+    getReleasedExercises(span: number): Observable<number> {
```

Comment:
`getNumberOfReleasedExercises` and so on :)

Reply:
Yes, sorry, this is also deprecated code which will be refactored after this PR and ist just here for semantic purposes.